test/interface/static_array_tests.jl

```diff
@@ -9,7 +9,7 @@ ode = ODEProblem(f, u0, (0.,1.))
 sol = solve(ode, Euler(), dt=1e-2)
 sol = solve(ode, Tsit5())
 
-u0 = fill(zero(SVector{2,Float64}), 2) .+ 1
+u0 = fill(zero(SVector{2,Float64}), 2)
 u0[1] = ones(SVector{2,Float64}) .+ 1
```

Comment:
This leads to a different initialization than before, it seems? Maybe it would be easier to just use:

```suggestion
u0 = [fill(2, SVector{2,Float64}), ones(SVector{2,Float64})]
```

Reply:
the values may be irrelevant in this test.

Reply:
A solution with a component that is zero constantly did not seem useful but probably it does not matter here and I noticed that the test with `MVector` above also uses a zero component. However, even in this case I'd say the following initialization
```suggestion
u0 = [fill(2, SVector{2,Float64}), zeros(SVector{2,Float64})]
```
is more natural (similarly, the example above could be changed).

Reply:
I think the " .+ 1" it was just a typo, which did not any harm as long as `+(::SArray, ::Number)` was defined.

Reply:
No, it was completely intentional. The tests were added a long time ago, I assume when only `zeros` but not `ones` was defined for StaticArrays. Initially, it was even just `fill(...) + 1` (without the broadcast) since this was valid syntax back then.